mods/mesecons/mesecons_luacontroller/init.lua

```diff
@@ -205,18 +205,6 @@ local function safe_print(param)
 	string_meta.__index = sandbox -- Restore string sandbox
 end
 
-local function safe_serlialize(param)
```

Comment:
This is used by some things on the server, though I think it might've been updated on the newer hostings of the server, and I guess you can send tables with digiline_send anyways so it wasn't necessary in the first place, so it's ok.

Reply:
I cannot find things using this function.